Packs/CommonDashboards/.pack-ignore

```diff
@@ -6,3 +6,6 @@ ignore=BA101
 
 [file:dashboard-SLA.json]
 ignore=BA101
+
+[file:1_4_4.md]
+ignore=RN113,RN114
```

Comment:
since it is a new script it fails on:
`[2024-06-06 10:29:46,125] [ERROR] Packs/CommonDashboards/ReleaseNotes/1_6_1.md: [RN114] - The script "UseCaseAdoptionMetrics" does not exist in the "CommonDashboards" pack.
Please use "demisto-sdk update-release-notes -i Packs/CommonDashboards"`

https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/jobs/6389704
is there another way to solve it? it is a new script